ansible/pbTestScripts/testScript.sh

```diff
@@ -4,29 +4,104 @@ set -u
 branchName='NULL'
```

Comment:
Why `NULL` as the default here instead of `''` which is used for everything else?